electrum/gui/qml/components/WalletMainView.qml

```diff
@@ -52,7 +54,7 @@ Item {
                 app.stack.push(Qt.resolvedUrl('TxDetails.qml'), { rawtx: data })
             } else if (Daemon.currentWallet.isValidChannelBackup(data)) {
                 var dialog = app.messageDialog.createObject(app, {
-                    title: qsTr('Import Channel backup?'),
+                    title: qsTr('Import Channel Backup?'),
```

Comment:
is this spelling change worth it to nuke the translations? :P

Reply:
I was doubting whether to do it, but decided to go ahead for consistency (although there's the plural form left still, e.g. 'Channel backups'), and include it in this PR to be reviewed :)